src/components/card/card.js

```diff
@@ -8,7 +8,8 @@
 angular.module('material.components.card', [
     'material.core'
   ])
-  .directive('mdCard', mdCardDirective);
+  .directive('mdCard', mdCardDirective)
+  .directive('mdPlainImage', mdPlainImageDirective);
```

Comment:
If we don't mind introducing a breaking change, then I think your `<md-card-image>` solution is great. I just wanted to make sure we knew it was a breaking change to do something like that 😄